src/EndOfYearBanner/canRender.ts

```diff
@@ -3,9 +3,9 @@ import type { BrazeMessageProps } from './index';
 /** These are in a seperate file to enable tree shaking of the logic deciding if a Braze message can be rendered
  * this means the user won't download the Braze components bundle when the component can't be shown.
  */
-export const COMPONENT_NAME = 'TheGuardianIn2020Banner';
+export const COMPONENT_NAME = 'EndOfYearBanner';
 
 export const canRender = (brazeMessageProps: BrazeMessageProps): boolean => {
-    const { header, body } = brazeMessageProps;
-    return Boolean(header && body);
+    const { header, body, buttonText, buttonUrl } = brazeMessageProps;
+    return Boolean(header && body && buttonText && buttonUrl);
```

Comment:
Should we be checking here for `imageUrl`? The docs say it's required (I think we said a default didn't make sense?)

Reply:
I believe the `Appbanner` `canRender` also does some checks to verify that the image host etc looks correct, might be worth checking and aligning with how we handle things there.

Reply:
Yep I agree. I've added this i.guim check to this banners canRender now.